app/components/agenda/agenda-overview.js

```diff
@@ -12,18 +12,15 @@ export default class AgendaOverview extends AgendaSidebar {
    * @argument currentAgenda
    * @argument previousAgenda
    */
-  @service sessionService;
-  @service agendaService;
 
-  @service('current-session') currentSessionService;
+  @service currentSession;
 
   dragHandleClass = '.ki-drag-handle-2';
 
   @tracked isEditingOverview = null;
-  @tracked showLoader = null;
 
   get isDraggingEnabled() {
-    return this.currentSessionService.isEditor && this.isDesignAgenda;
+    return this.currentSession.isEditor && this.isDesignAgenda;
```

Comment:
note to self, where does `this.isDesignAgenda` come from ? not via args, no local getter

Reply:
ah, `extends AgendaSidebar`...